tools/include/eosio/codegen.hpp

```diff
@@ -273,6 +340,26 @@ namespace eosio { namespace cdt {
                cg.notify_handlers.insert(full_notify_name); // insert the method action, so we don't create the dispatcher twice
             }
 
+            // We allow a method to be tagged as both `action` and `call`
+            if (decl->isEosioCall()) {
+               name = generation_utils::get_call_name(decl);
+               validate_name(name, [&](auto s) {
+                  CDT_ERROR("codegen_error", decl->getLocation(), std::string("call name (")+s+") is not a valid eosio name");
+               });
+
+               if (!_call_set.count(name))
+                  _call_set.insert(name);
+               else {
+                  auto itr = _call_set.find(name);
+                  CDT_CHECK_ERROR(*itr == name, "codegen_error", decl->getLocation(), "call declaration doesn't match previous declaration");
```

Comment:
This check can never fail. The `else` is because `count()` returned `>0`. Also if `find` didn't find `name` it would return `end()` which is not valid to `*itr`.

The same is true for the similar code of `_action_set` above.